# HG changeset patch
# User PrasannaKumar Muralidharan <prasannatsmkumar@gmail.com>
# Date 1384448503 -19800
#      Thu Nov 14 22:31:43 2013 +0530
# Node ID 265f9186e543aa02c05d52bbc2f4979d2d2ba2a4
# Parent  1ed269bdea585c35766292b54c8943959f5194d7
Fix uninitialised parameters in pixel2status

* __init_fltk__.cc (pixel2status): Uninitialised variables are processed so
initialise them correctly

diff --git a/libinterp/dldfcn/__init_fltk__.cc b/libinterp/dldfcn/__init_fltk__.cc
--- a/libinterp/dldfcn/__init_fltk__.cc
+++ b/libinterp/dldfcn/__init_fltk__.cc
@@ -1081,16 +1081,17 @@ private:
   {
     pixel2status (gh_manager::get_object (ax), px0, py0, px1, py1);
   }
 
   void pixel2status (graphics_object ax, int px0, int py0,
                      int px1 = -1, int py1 = -1)
   {
     double x0, y0, x1, y1;
+    x0 = y0 = x1 = y1 = octave_NaN;
     std::stringstream cbuf;
     cbuf.precision (4);
     cbuf.width (6);
     pixel2pos (ax, px0, py0, x0, y0);
     cbuf << "[" << x0 << ", " << y0 << "]";
     if (px1 >= 0)
       {
         pixel2pos (ax, px1, py1, x1, y1);
